app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,63 @@
+class PasswordResetsController < ApplicationController
+  before_action :load_user, only: [:edit, :update, :create]
```

Comment:
chuyển qua dùng %i nha
```suggestion
  before_action :load_user, only: %i(edit update create)
```